docs/en/core-services/Caching.md

```diff
@@ -129,24 +128,21 @@ For example, users often specify the following tiers:
 
 #### Writing Data
 
-When a user writes a new block, it is written to the top tier by default. If eviction cannot free up
-space in the top tier, the write will fail. If the file size exceeds the size of the top tier, the
-write will also fail.
+When a user writes a new block, it is written to the top tier by default. If the top tier does not have enough free space,
+then the next tier is tried. If no space is found on all tiers, Alluxio frees up space for new data as its storage is designed to be volatile.
+Eviction will start attempting to evict blocks from the worker, based on the block annotation policy.
```

Comment:
either saying "block annotation policy (explained later)" or put a hyperlink pointing to the section, since this is the first time we introduce this concept.